src/app/hearing-part/utils/hearing-utils.ts

```diff
@@ -0,0 +1,9 @@
+import { ListingCreate } from '../models/listing-create';
+
+export function isMultiSessionListing(listing: ListingCreate): boolean {
```

Comment:
Maybe it could be put into ListingCreate model's file if it relates to that model?

Reply:
true